WordPressKit/PlanServiceRemote.swift

```diff
@@ -3,84 +3,150 @@ import WordPressShared
 import CocoaLumberjack
 
 public class PlanServiceRemote: ServiceRemoteWordPressComREST {
-    public typealias SitePlans = (activePlan: RemotePlan?, availablePlans: [RemotePlan])
+    public typealias AvailablePlans = (plans: [RemoteWpcomPlan], groups: [RemotePlanGroup], features: [RemotePlanFeature])
+
+    typealias EndpointResponse =  [String: AnyObject]
```

Comment:
I'm not in love with this typealias (not my style ;P), but I see how it clears things up down the line. In any case, there's double space after the `=` :)

Reply:
Eagle eyes right there.